dracut/30ignition/ignition-reboot.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Ignition (reboot)
```

Comment:
I'd avoid using the `Ignition (X)` for things that aren't Ignition stages.

Reply:
Changed to `Description=Reboot after Ignition to apply kargs`